static/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "RuneScraper",
+  "version": "1.0.0",
+  "description": "Web application to pull together various data points to assist the RuneScape community",
+  "main": "index.js",
+  "repository": "git@github.com:benrandall/idb.git",
+  "author": "Edward DeCoste <elmdecoste@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.9",
+    "node-sass": "^4.7.2",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.20.1",
+    "webpack": "^3.11.0"
+  },
+  "scripts": {
+    "build": "webpack -p --progress --config webpack.config.js",
+    "dev-build": "webpack --progress -d --config webpack.config.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --progress -d --config webpack.config.js --watch"
+  },
+  "babel": {
+    "presets": [
+      "es2015",
```

Comment:
Any reason to use es2015 over es6?

Reply:
Not really, we can use ES6 if we want to

Reply:
Kk fine for now we can use es6 if we want later, adds some cool typing